requirements.txt

```diff
@@ -1,3 +1,4 @@
 click>=8.1.0,<8.2.0
 requests>=2.0.0,<3.0.0
-PyYAML
\ No newline at end of file
+PyYAML
+graphql-core==3.2.3
```

Comment:
Does this need to be pinned to a specific version or should it be >= like the other requirements here?